File: javascript/features/support/doc-string-steps-example.js
const { Given, When, Then } = require('cucumber')

Given('I have filled in the address details form', function () {
    // Given glue code here
});

When('I submit the form', function () {
    // When glue code here
});


Then('the address is processed and I receive the message', function (docString) {
    console.log(docString);
});

File: java/src/test/java/stepdefinitions/DocStringStepsExample.java
package stepdefinitions;

import cucumber.api.java.en.Given;
import cucumber.api.java.en.Then;
import cucumber.api.java.en.When;

public class DocStringStepsExample {

    @Given("I have filled in the address details form")
    public void fillInForm() {
        // Given glue code in here
    }

    @When("I submit the form")
    public void submitForm() {
        // When glue code in here
    }

    @Then("the address is processed and I receive the message")
    public void docStringDemo(String docString) {
        System.out.println(docString);
    }

}
